conans/client/build/autotools_environment.py

```diff
@@ -146,11 +147,18 @@ def configure(self, configure_dir=None, args=None, build=None, host=None, target
         if self._conanfile.package_folder is not None:
             if not args:
                 args = ["--prefix=%s" % self._conanfile.package_folder.replace("\\", "/")]
-            elif not any(["--prefix=" in arg for arg in args]):
+            elif not self._is_flag_in_args("prefix", args):
                 args.append("--prefix=%s" % self._conanfile.package_folder.replace("\\", "/"))
-
-            if not any(["--libdir=" in arg for arg in args]):
-                args.append("--libdir=${prefix}/lib")
+            for varname in ["bindir", "sbin", "libexec"]:
+                if not self._is_flag_in_args(varname, args):
+                    args.append("--%s=${prefix}/%s" % (varname, DEFAULT_BIN))
```

Comment:
does it mean `--bindir` will now be always added to the configure command line?
I worry about libraries which provide autotools-like configure scripts, which are not really auto-tools based (e.g. ffmpeg, libvpx, etc.)
at least there should be way to avoid adding these arguments from recipe (e.g. like declaring build/host/target to None)

Reply:
I think defining the output folders is something mandatory for the build helpers and this behavior should no rely on the platform where the recipe is being build. Will have to aware users that this might be breaking for the cases you pointed out. i will have a look to see if there is any way to opt-out

Reply:
yes, opt-out would be nice to have, at least to let hand-crafted configure scripts to continue working in conan environment.

Reply:
Yes, please opt-out in the build helper.